src/Insurello.AsyncExtra/AsyncExtra.fs

```diff
@@ -60,3 +60,15 @@ module AsyncResult =
             asyncs
             |> List.fold folder (fromResult (Ok []))
             |> map List.rev
+
+    type AsyncResultBuilder() =
```

Comment:
Nice, but some test using CE would be nice

Reply:
I agree! I just didn't have time :P